CHANGES.md

```diff
@@ -144,6 +144,22 @@ OpenSSL 3.5
 
    *Zhiguo Zhou, Wangyang Guo (Intel Corp)*
 
+ * Fix EVP_DecodeUpdate(): do not write padding zeros to the decoded output.
+
+   According to the documentation,
+   for every 4 valid base64 bytes processed (ignoring whitespace, carriage returns and line feeds),
+   EVP_DecodeUpdate() produces 3 bytes of binary output data
+   (except at the end of data terminated with one or two padding characters).
+   However, the function behaved like an EVP_DecodeBlock():
+   produces exactly 3 output bytes for every 4 input bytes.
+   Such behaviour could cause writes to a non-allocated output buffer
+   if a user allocates its size based on the documentation and knowing the padding size.
+
+   The fix makes EVP_DecodeUpdate() produce
+   exactly as many output bytes as in the initial non-encoded message.
+
+   *Valerii Krygin*
```

Comment:
More important than updating the `CHANGES.md` is to update `doc/man3/EVP_EncodeInit.pod` which describes the current behaviour of `EVP_DecodeBlock()`, and note the change from previous behaviour in the HISTORY section.
Best to run `make doc-nits` before pushing the updated docs, resolving any reported issues.

Reply:
Got it. I updated the documentation and the copyright years.